src/mps/mpo.jl

```diff
@@ -558,6 +569,18 @@ function Tensors.truncate!(M::Union{MPS,MPO}; kwargs...)
 
 end
 
+function Base.:*(x::Number,M::Union{MPS,MPO})
+    N = deepcopy(M)
+    N.A_[div(size(N), 2)] = N.A_[div(size(N), 2)] .* x
```

Comment:
Any reason not to use N[div(size(N),2)] here instead of N.A_? Generally we'd like to avoid directly accessing data members unless necessary.